src/libs/actions/BankAccounts.js

```diff
@@ -81,7 +81,7 @@ function getVBBADataForOnyx() {
                 value: {
                     isLoading: false,
                     errors: {
-                        [DateUtils.getMicroseconds()]: Localize.translateLocal('paymentsPage.addBankAccountFailure'),
```

Comment:
We're passing `'paymentsPage.addBankAccountFailure'` to the utility method, but all it's doing is storing that key as the error instead of translating it. So this is what the error looks like:

![Screenshot 2023-06-02 at 10 04 35 AM](https://github.com/Expensify/App/assets/22219519/6fc15a92-46ee-4316-969d-4c1be3e7b872)

but it should be:

`1685721717986000: "An unexpected error occurred while trying to add your bank account. Please try again."`

Reply:
We're only calling `translateIfPhraseKey` in DotIndicator, but there are other places displaying the error text, e.g. Forms, that don't use DotIndicator.